src/qbe_ir_generator.cpp

```diff
@@ -37,6 +38,50 @@ void VWriteOut(fmt::string_view format, fmt::format_args args) {
   fmt::vprint(output, format, args);
 }
 
+std::string GetBinaryOperator(BinaryOperator op) {
+  switch (op) {
+    case BinaryOperator::kAdd:
+      return "add";
+    case BinaryOperator::kSub:
+      return "sub";
+    case BinaryOperator::kMul:
+      return "mul";
+    case BinaryOperator::kDiv:
+      return "div";
+    case BinaryOperator::kMod:
+      return "rem";
+    // TODO: update comparison instructions with no data type, such as "gt".
+    case BinaryOperator::kGt:
+      return "csgtw";
+    case BinaryOperator::kGte:
+      return "csgew";
+    case BinaryOperator::kLt:
+      return "csltw";
+    case BinaryOperator::kLte:
+      return "cslew";
+    case BinaryOperator::kEq:
+      return "ceqw";
+    case BinaryOperator::kNeq:
+      return "cnew";
+    default:
+      return "Unknown";
```

Comment:
We are returning an "unknown" type that does not start with a capital letter for our types:
https://github.com/fruits-lab/VitaminC/blob/4d6a6960ce60c032406b6f5e08d8616b3759ac4d/src/type.cpp#L5-L13

Do you think it's better if we maintain consistency in this regard? Alternatively, should we throw an error (or trigger an assertion failure) here, since `ExprType::kUnknown` is valid, but operators are not?

Reply:
As we discussed, we will change unknown operator to unreachable in the next PR.